ilib-webpack-plugin.js

```diff
@@ -448,11 +448,21 @@ function IlibDataPlugin(options) {
 }
 
 IlibDataPlugin.prototype.apply = function(compiler) {
-    compiler.plugin('compilation', function(compilation, callback) {
+    // webpack 4.X or 3.X?
+    var compilationHook = (typeof(compiler.hooks) !== "undefined") ?
```

Comment:
You might be able to leverage https://webpack.js.org/api/loaders/#thisversion to make this more future proof

Reply:
Yeah, that’s for loaders instead of plugins
I ran the whole thing in the debugger and examined everything available in the “compiler” and the “compilation” objects, and there was no mention of any version anywhere
I guess they forgot it…